### ISSUE TITLE
Formatting of @ directives within tag attributes broken?

### ISSUE THREAD

**Issue by cheesegrits** [Stillat/blade-parser-typescript#67]
It seems to be struggling with [at]directives inside tag attributes?

This by itself is fine, so ...

```
@unless(config('filament-media-library.settings.show-upload-box-by-default'))
    x-show="showUploadBox"
    x-collapse
    x-cloak
@endunless
```

... formats to ...

```
        @unless (config('filament-media-library.settings.show-upload-box-by-default'))
            x-show="showUploadBox" x-collapse x-cloak
        @endunless
```

... but ...

```
<div
    class="my-4"
    @unless(config('filament-media-library.settings.show-upload-box-by-default'))
         x-show="showUploadBox"
         x-collapse
         x-cloak
     @endunless
>
```

... formats to ...

```
<div
    class="my-4"
    @unless(config('filament-media-library.settings.show-upload-box-by-default'))
        x-show="showUploadBox"
    x-collapse
    x-cloak
    @endunless
>
```

Or ...

```
<div
    class="my-4"
    @unless(config('filament-media-library.settings.show-upload-box-by-default')) x-foo
         x-show="showUploadBox"
         x-collapse
         x-cloak
     @endunless
>
```

... (with the errant x-foo trailing) formats to ...

```
<div
    class="my-4"
    @unless(config('filament-media-library.settings.show-upload-box-by-default')) x-foo
    x-show="showUploadBox"
    x-collapse
    x-cloak
    @endunless
>
```

Thanks for all your great work on this plugin.

**Comment by JohnathonKoster** [comment]
Not broken necessarily - just a current limitation since the underlying parsers weren't good enough at the time of initial release to confidently do this - so it works to return the original content to not risk breaking things.

Will look into it now, though :)

**Comment by cheesegrits** [comment]
Thanks!

**Comment by JohnathonKoster** [comment]
This feature is now available in `1.6.8` 🙂
